mlflow/tracking/client.py

```diff
@@ -469,6 +471,43 @@ def search_registered_models(
         :return: A PagedList of :py:class:`mlflow.entities.model_registry.RegisteredModel` objects
                 that satisfy the search expressions. The pagination token for the next page can be
                 obtained via the ``token`` attribute of the object.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow
+
+            model_name="CordobaWeatherForecastModel"
+            filter = "name='{}'".format(model_name)
```

Comment:
```suggestion
            filter_string = "name='{}'".format(model_name)
```

because `filter` conflicts with python's built-in `filter` function.

Reply:
good point. I'll change it!